src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -226,7 +226,7 @@
             {% set label = name|humanize %}
         {%- endif -%}
     {%- endif -%}
-    <button type="{{ type|default('button') }}" {{ block('button_attributes') }}>{{ translation_domain is same as(false) ? label : label|trans({}, translation_domain) }}</button>
+    <button type="{{ type|default('button') }}" {{ block('button_attributes') }}>{{ (translation_domain is same as(false) ? label : label|trans({}, translation_domain))|raw }}</button>
```

Comment:
IMO that's a BC break (and it's also inconsistent with other places where we output the label). If you need this in your project, I suggest to override the `button_widget` block.